WebDevStudios/wd_s
# Move build packages to dependencies

Closes #970

### DESCRIPTION

Moved packages from devDependencies to dependencies

Removed postinstall script which will fail in buddy. Renamed it to `npm run build:all`

### OTHER


### STEPS TO VERIFY

1. Check out this branch
2. run `npm install --production && npm run build`
3. observe no errors

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?
